clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_vector_daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: vector-audit-log
+  namespace: api-audit-log
+  labels:
+    k8s-app: vector-audit-log
+spec:
+  selector:
+    matchLabels:
+      k8s-app: vector-audit-log
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        k8s-app: vector-audit-log
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      dnsPolicy: Default
+      containers:
+      - name: vector-audit-log
+        image: docker.io/timberio/vector:0.8.2-alpine
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 500m
+            memory: 700Mi
```

Comment:
Thats a lot, is this really needed?

Reply:
Good point.
I copied it from 

https://github.com/openshift/release/blob/a3ca62c5732160d677f08c41c8d09db1811654fe/core-services/cluster/logging/admin_fluentd-daemonset.yaml#L49-L52

Certainly api audit log shouldnt be as many as all container logs.
Let us run it sometime and get some numbers to tune.

I will leave this unsolved for now.

Reply:
https://grafana-openshift-monitoring.apps.build01.ci.devcluster.openshift.com/d/85a562078cdf77779eaa1add43ccec1e/kubernetes-compute-resources-namespace-pods?orgId=1&refresh=10s&var-datasource=prometheus&var-cluster=&var-namespace=api-audit-log&var-interval=4h&from=now-24h&to=now

![Screen Shot 2020-04-13 at 12 07 37 PM](https://user-images.githubusercontent.com/4013349/79136710-74ac7800-7d7f-11ea-9270-4a589bea47ae.png)

Reply:
Changed to smaller numbers